cpp/ql/lib/semmle/code/cpp/security/boostorg/asio/protocols.qll

```diff
@@ -394,14 +394,19 @@ module BoostorgAsio {
    * Constructs a standard data flow computation for protocol values to the first argument
    * of a context constructor.
    */
-  module SslContextCallMake<SslContextCallConfigSig Config> {
+  module SslContextCallGlobal<SslContextCallConfigSig Config> {
     private module C implements DataFlow::ConfigSig {
       import Config
     }
 
     import DataFlow::Global<C>
   }
 
+  /** DEPRECATED: Use `SslContextCallGlobal` instead. */
+  deprecated module SslContextCallMake<SslContextCallConfigSig Config> {
```

Comment:
If you're able to edit the change note that talks about the introduction of this name, then it appears that you might have been fast enough to not need this deprecation, i.e. if it hasn't been released then no need to deprecate it.

Reply:
Good point. I was maybe a bit too careful here.